### PR TITLE
#[ 158463674] Include calendar Id in UpdateRoom mutation

### PR DESCRIPTION
**Pull request by dkam26**
#### What does this PR do?

-  Include calendar Id in UpdateRoom mutation

#### Description of Task to be completed?

- Including the calendar ID in the Updating room mutation 

#### What are the relevant pivotal tracker stories?

- #158463674

#### Screenshots

 - Current rooms in the database

screen shot 2018-06-19 at 11 14 52 pm
- Lets update the capacity of Mbarara meeting room

screen shot 2018-06-19 at 11 26 54 pm
- We can see the changes with calendar_id field

screen shot 2018-06-20 at 7 35 45 am

### CODE REVIEW

File: api/room/schema.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class Arguments:
         room_type = graphene.String()
         capacity = graphene.Int()
         image_url = graphene.String()
+        calendar_id = graphene.String()
```
Comment (shemogumbe) [inline api/room/schema.py]: LGTM

Comment (shemogumbe) [reply api/room/schema.py]: Though I see no reason why changes on revisions are on this commit

File: api/room/schema.py [hunk 2]
```diff
@@ -54,12 +55,29 @@ def mutate(self, info, room_id, **kwargs):
...
 class Query(graphene.ObjectType):
     all_rooms = graphene.List(Room)
     get_room_by_id = graphene.Field(
         Room,
         room_id=graphene.Int()
-        )
+    )
     room_schedule = graphene.Field(
         Calendar,
         calendar_id=graphene.String(),
```
Comment (PatrickCmd) [inline api/room/schema.py]: I see that this room_schedule field is declared twice on your branch. Please check and remove one of the fields.

Comment (dkam26) [reply api/room/schema.py]: @PatrickCmd ,Thanks for correction.All is well now

**Pull request merged**